### DISCUSSION TITLE
Can we expect Dart support in Functions?

### DISCUSSION THREAD

**Discussion by tananga** [supabase/supabase#3885] [Questions]
Can we expect Dart support in Functions? This is not supported by Firebase and Im very angry about it. I can pay extra just please add it to the scope. As a flutter Developer Dart Functions is must have for me.

**Comment by J0** [comment]
Hey! 

Thank you for your patience -- we hear your concerns. Dart/Flutter is  important to us and the client SDK is maintained by the Supabase team. We currently support [edge functions in our Dart library](https://supabase.com/docs/reference/dart/invoke) 

Do let us know if you face any issues while using.

Thanks!

**Reply by tananga** [reply]
OK But I was thinking more about write my edge function in Dart instead of being able to connect from dart to Functions written in typescript. Is it gonna be possible to code my function in dart?
